### PR TITLE
Added buttons that can delete assets

### LINKED ISSUE #157: UI does not allow deleting assets
**Describe the bug (or missing functionality)**

Currently, new assets can be registered using the MLX UI. Existing assets can be un-published and un-featured to effectively disappear, but there is no capability to delete registered assets (for admins).

@drewbutlerbb4

### PR DESCRIPTION
**Pull request by Zachary-Fernandes**
The UI now includes buttons when accessing all the assets. These
buttons delete the assets they are associated with when they are
clicked.

Resolves #157

@ckadner

### CODE REVIEW

**Comment by ckadner** [comment]
Thank you @Zachary-Fernandes -- could you add a screenshot as well?

**Comment by Zachary-Fernandes** [comment]
Sure thing, I can demonstrate it by deleting Project CodeNet Language Classifier.

Here it is before deleting:
beforedelete

Here it is after deleting:
afterdelete

File: dashboard/origin-mlx/src/pages/MetaAllPage.tsx [hunk 1]
```diff
@@ -236,6 +237,20 @@ function MetaAllPage(props: MetaAllPageProps) {
...
+                      className="delete-button"
+                      variant="contained"
+                      onClick={async function() {
+                        await fetch(`${API}/apis/v1alpha1/${asset.type}/${asset.id}`, {
+                          method: 'DELETE'
+                        });
+                        window.location.reload();
+                       }}
+                      >
+                      <span className="delete-button-text">DELETE</span>
```
Comment (ckadner) [inline dashboard/origin-mlx/src/pages/MetaAllPage.tsx:251]: I would make it Title-case: `Delete`, and update the  `className="delete-button-text"` to a color and font that matches the rest of this page

File: dashboard/origin-mlx/src/pages/MetaAllPage.tsx [hunk 2]
```diff
@@ -66,6 +66,7 @@ function MetaAllPage(props: MetaAllPageProps) {
     { id: 'feat', label: 'Featured', numeric: false },
     { id: 'pub', label: 'Published', numeric: false },
     { id: 'cat', label: tagName, numeric: false },
+    { id: 'del', label: 'Delete Asset', numeric: false },
```
Comment (ckadner) [inline dashboard/origin-mlx/src/pages/MetaAllPage.tsx:69]: just `'Delete'` is enough

**Review by ckadner — CHANGES_REQUESTED** [review]
Thanks @Zachary-Fernandes -- just a few nit-picks ;-)

**Comment by ckadner** [comment]
@Zachary-Fernandes does this screenshot reflect your latest changes?

image

**Comment by Zachary-Fernandes** [comment]
Yes. It originally was red with a minus sign because I could not figure out how to spawn lowercase text, but after making a new style in Models.css, I figured out how to do this.

**Review by ckadner — APPROVED** [review]
/lgtm

Thank you @Zachary-Fernandes

**Pull request merged**